common/helpers/picasso.js

```diff
@@ -275,26 +279,39 @@ export function positionPercentage(milliseconds, times) {
 export function filterScheduleEntriesByDay(scheduleEntries, day, times) {
   return scheduleEntries.filter((scheduleEntry) => {
     return (
-      utcStringToTimestamp(scheduleEntry.start) < dayEndTimestamp(day, times) &&
-      utcStringToTimestamp(scheduleEntry.end) > dayStartTimestamp(day, times)
+      dayjs.utc(scheduleEntry.start).isBefore(dayEnd(day, times)) &&
+      dayjs.utc(scheduleEntry.end).isAfter(dayStart(day, times))
     )
   })
 }
 
 /**
- * Calculates the visible day start timestamp
+ * @param day a day object of the api
```

Comment:
Maybe keep the prose descriptions from before?

Reply:
done